### DISCUSSION TITLE
Is there a simple example program using Python to operate a model?

### DISCUSSION THREAD

**Discussion by zhi8617** [mallman/CoreMLaMa#3] [General]
I haven't learned the syntax of Swift, so understanding it is quite challenging. Thank you

**Comment by mallman** [comment]
Hi @zhi8617. I don't have a Python example, but I know that Apple's Core ML Tools package has a Python API for running a Core ML model on a Mac. Have you had a look at their documentation? Check it out: https://apple.github.io/coremltools/docs-guides/source/model-prediction.html
